frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -310,9 +310,15 @@ end
 
 function ReaderBookmark:onToggleBookmark()
     self:toggleBookmark()
-    self.view.footer:onUpdateFooter(self.view.footer_visible)
     self.view.dogear:onSetDogearVisibility(not self.view.dogear_visible)
-    UIManager:setDirty(self.view.dialog, "ui")
+    -- Refresh the dogear first, because it might inherit ReaderUI refresh hints.
+    UIManager:setDirty(self.view.dialog, function()
+        return "ui",
+        self.view.dogear:getRefreshRegion()
+    end)
+    -- And ask for a footer refresh, in case we have bookmark_count enabled.
+    -- Assuming the footer is visible, it'll request a refresh regardless, but the EPDC should optimize it out if no content actually changed.
```

Comment:
Is that true for older boards ? (ie. my GloHD :))

Reply:
Even if the driver doesn't actually optimize it out, even GC16 won't visually affect unchanged pixels, and that's been the case forever ;).